sdks/java/harness/src/main/java/org/apache/beam/fn/harness/FnHarness.java

```diff
@@ -206,18 +205,16 @@ public static void main(
 
       LoadingCache<String, BeamFnApi.ProcessBundleDescriptor> processBundleDescriptors =
           CacheBuilder.newBuilder()
+              .maximumSize(1000)
+              .expireAfterAccess(10, TimeUnit.MINUTES)
```

Comment:
Is this too large of a time interval? I'm assuming the SDK would ideally not wait ~10 min before starting the bundle, right?

Reply:
This is the time duration between never seeing a bundle of this type and discarding its description from the cache. It could be smaller, but I figured it was good to start out very conservatively (both count and time).